stdlib/test/builtin/test_tuple.mojo

```diff
@@ -0,0 +1,27 @@
+# ===----------------------------------------------------------------------=== #
+# Copyright (c) 2024, Modular Inc. All rights reserved.
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions:
+# https://llvm.org/LICENSE.txt
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ===----------------------------------------------------------------------=== #
+# RUN: %mojo %s
+
+from testing import assert_true, assert_false
+
+
+def test_tuple_contains():
+    var a = (1, True)
```

Comment:
Can you please change this to not have `1` (or) `0` in it? Since Bool would convert to that implicitly, it can get tricky if something goes wrong with the call emission.